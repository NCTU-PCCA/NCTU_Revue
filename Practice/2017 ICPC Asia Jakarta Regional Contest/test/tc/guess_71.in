14395546605009268 54
<<>><<<><>><<<>><><<<<><<>>>><<>>><<><><>><>><>><<>><>
