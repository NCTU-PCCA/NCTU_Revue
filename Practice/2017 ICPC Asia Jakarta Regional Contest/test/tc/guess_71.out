14395546605009268 14395546605009267 1 2 14395546605009266 14395546605009265 14395546605009264 3 14395546605009263 4 5 14395546605009262 14395546605009261 14395546605009260 6 7 14395546605009259 8 14395546605009258 14395546605009257 14395546605009256 14395546605009255 9 14395546605009254 14395546605009253 10 11 12 13 14395546605009252 14395546605009251 14 15 16 14395546605009250 14395546605009249 17 14395546605009248 18 14395546605009247 19 20 14395546605009246 21 22 14395546605009245 23 24 14395546605009244 14395546605009243 25 26 14395546605009242 27
